src/app/shared/issue-tables/issue-tables.component.ts

```diff
@@ -44,7 +45,8 @@ export class IssueTablesComponent implements OnInit, AfterViewInit {
               private githubService: GithubService,
               private issueService: IssueService,
               private phaseService: PhaseService,
-              private errorHandlingService: ErrorHandlingService) { }
+              private errorHandlingService: ErrorHandlingService,
+              private loggingService: LoggingService) { }
 
   ngOnInit() {
```

Comment:
Should we add a logging statement here as well just for clarity? Or do you think it's not necessary

Reply:
I think it might be easier for us to spot which component the logging statements are for!

Reply:
Like that the logs are coming from the `IssueTablesComponent`?

Reply:
Yup, so when it is initialised it will say "IssueTablesComponent has been initialised", and we know the following logging statements are for `IssueTablesComponent`! Not sure if it is necessary.

Reply:
Hmm I was thinking we can just `ctrl + f` can also find where the log 😅 But maybe instead initialization of that we can put like `IssueTablesComponent: ...` to indicate which message is from which component?

Reply:
That works as well!👍